.github/workflows/analyze-istio.yaml

```diff
@@ -0,0 +1,30 @@
+name: analyze-istio
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+
+jobs:
+  check-istio:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Get Istioctl
+        run: |
+          ISTIO_VERSION=$(yq eval '. | select(.kind == "Deployment") | .spec.template.spec.containers[0].image | split(":") | .[1]' ./istio/operator/manifests.yaml)
```

Comment:
Please create two steps, one that downloads the binary and another one that run the validation.

Reply:
It took an embarrassing number of iterations, but this is done.